build-support/bin/generate_builtin_lockfiles.py

```diff
@@ -128,6 +129,7 @@ class JvmTool(Tool[JvmToolBase]):
             PythonTool(Yamllint, "pants.backend.experimental.tools.yamllint"),
             PythonTool(Yapf, "pants.backend.python.lint.yapf"),
             PythonTool(Ruff, "pants.backend.experimental.python.lint.ruff"),
+            PythonTool(Pytype, "pants.backend.python.typecheck.pytype"),
```

Comment:
This list looks sorted (with the exception of Ruff). My autistic brain would like to keep it sorted so if you don't mind fixing that (including moving the Ruff line) that would be 🎉 in my 📖 :)

Reply:
hehe, done :)